setup.py

```diff
@@ -8,24 +8,6 @@
     version="0.1.1",
     description="A lightweight python module to interact with atlases for systems neuroscience",
     install_requires=requirements,
-    extras_require={
```

Comment:
Have these been moved anywhere? They weren't just for atlas generation, but also to allow contributors to easily set up a development environment (with pytest etc.).

Reply:
Ugh right, sorry! I was just thinking about the atlas generation. I'll move those backs